pkg/queue/queue.go

```diff
@@ -74,15 +74,25 @@ type ClusterQueue struct {
 	heap heapImpl
 }
 
-func newClusterQueue(cq *kueue.ClusterQueue) *ClusterQueue {
+func newClusterQueue(cq *kueue.ClusterQueue) (*ClusterQueue, error) {
+	var less lessFunc
+
+	// TODO(#10 denkensk): add a simple framework to support different queueing strategies
```

Comment:
this todo is not necessary

Reply:
we need one, but having a comment in the code isn't the right place to track that requirement :)